### PR TITLE
Make spelling of "initialize" consistent

### PR DESCRIPTION
**Pull request by fjeremic**
This will make searching easier as one does not have to check for
alternate spellings.

Signed-off-by: Filip Jeremic <fjeremic@ca.ibm.com>

### CODE REVIEW

**Comment by fjeremic** [comment]
This PR depends on https://github.com/eclipse/omr/pull/2490.

**Comment by fjeremic** [comment]
Jenkins test sanity plinux depends eclipse/omr#2490

**Comment by fjeremic** [comment]
The copyright failures all seem to be in the debugtools/DDR_Artifacts directories. These files do not even have copyright dates however, so this seems to be a tooling issue. @AdamBrousseau FYI. Perhaps the Copyright Check tool should validate that a copyright header exists in the first place before marking failures?

**Comment by AdamBrousseau** [comment]
I suppose we could ignore **Modified** files if they don't have any copyright, but we probably still want to fail if an **Added** file doesn't have a copyright. There is an open issue (#1043) to have a whitelist ability added to the tool, which would also solve this problem. Until the tool is rock solid, its mostly just a flag to point out "potential" issues. It's up to the committer to determine if the failure is a problem. Although I've seen the wrong decision made in the past.

**Comment by fjeremic** [comment]
@vijaysun-omr given that you're the committer for https://github.com/eclipse/omr/pull/2490 this one also needs a review. Both need to be merged together.

**Comment by 0xdaryl** [comment]
@fjeremic , would you mind rebasing please?

**Comment by fjeremic** [comment]
> @fjeremic , would you mind rebasing please?

@0xdaryl rebased boht OpenJ9 and OMR chnages.

Jenkins test sanity

**Comment by fjeremic** [comment]
> @fjeremic , would you mind rebasing please?

@0xdaryl rebased boht OpenJ9 and OMR chnages.

Jenkins test sanity depends eclipse/omr#2490

**Comment by keithc-ca** [comment]
Is this worth the churn given the ability to use regular expressions (e.g. `initiali[sz]e`)?

**Comment by fjeremic** [comment]
> Is this worth the churn given the ability to use regular expressions (e.g. initiali[sz]e)?

Perhaps a discussion to be had in https://github.com/eclipse/omr/issues/2486?

**Comment by keithc-ca** [comment]
FYI, none of the files below DDR_Artifacts need be changed: they're not consumed by current builds.
It may make sense to simply remove them (in another PR).

**Comment by fjeremic** [comment]
> FYI, none of the files below DDR_Artifacts need be changed: they're not consumed by current builds.
> It may make sense to simply remove them (in another PR).

I suppose if that's the case no point amending commits since we are removing it anyway. I'll leave it as is. When this is merged I'll open a PR to remove all the DDR_Artifacts.

**Comment by 0xdaryl** [comment]
Jenkins test sanity depends eclipse/omr#2490

**Comment by 0xdaryl** [comment]
Now that infra issues seem to be resolved, I re-ran the sanity tests just to be sure the 6 failures weren't hiding anything.

File: runtime/port/win32/j9sock.c [hunk 1]
```diff
@@ -158,7 +158,7 @@ findError (int32_t errorCode)
 {
 	switch (errorCode)
 	{
-		case WSANOTINITIALISED:
+		case WSANOTINITIALIZED:
```
Comment (0xdaryl) [inline runtime/port/win32/j9sock.c]: `WSANOTINITIALISED ` is actually a Windows socket error code.

https://msdn.microsoft.com/en-us/library/windows/desktop/ms740668(v=vs.85).aspx

**Comment by fjeremic** [comment]
Jenkins test sanity depends eclipse/omr#2490

File: runtime/port/win32/j9sock.c (hunk 1 above)
Comment (fjeremic) [reply runtime/port/win32/j9sock.c]: Fixed in 82cbf35. Launched a new build now. Thanks for catching this!

**Comment by fjeremic** [comment]
Windows failures are due to https://github.com/eclipse/openj9/issues/2129. @0xdaryl this one looks good to review again.

**Pull request merged**